server/src/test/java/io/crate/integrationtests/LookupJoinIntegrationTest.java

```diff
@@ -22,45 +22,68 @@
 package io.crate.integrationtests;
 
 
-
 import static io.crate.testing.Asserts.assertThat;
 
 import org.elasticsearch.test.IntegTestCase;
-import org.junit.Before;
 import org.junit.Test;
 
 import io.crate.testing.UseHashJoins;
 import io.crate.testing.UseRandomizedOptimizerRules;
 
 public class LookupJoinIntegrationTest extends IntegTestCase {
 
-    @Before
-    public void setup() throws Exception {
+    @UseRandomizedOptimizerRules(0)
+    @UseHashJoins(1)
+    @Test
+    public void test_lookup_join_with_two_collect_operators() throws Exception {
         execute("create table doc.t1 (id int) with(number_of_replicas=0)");
-        execute("insert into doc.t1 (id) select b from generate_series(1,10000) a(b)");
+        execute("insert into doc.t1 (id) select b from generate_series(1,20000) a(b)");
```

Comment:
Why the change here? Shouldn't the ratio remain the same? The test case is getting a bit expensive

Reply:
That was related to the filter in the query. It's fixed now. I adapted the query.